FirebaseMLModelDownloader/Tests/Integration/ModelDownloaderIntegrationTests.swift

```diff
@@ -162,6 +162,8 @@ final class ModelDownloaderIntegrationTests: XCTestCase {
       defaults: .createTestInstance(testName: #function),
       progressHandler: { progress in
         XCTAssertNotNil(progress)
```

Comment:
This `NotNil` line can be removed since the `progress` variable is not an Optional (it's a plain `Float`) so it can never fail.